### ISSUE TITLE
Shin Megami Tensei V - ingame graphics are white and green only in Vulkan on latest builds

### ISSUE THREAD

**Issue by tapperp** [yuzu-emu/yuzu#7563]
Using latest builds (i.e. 845, 844 etc.) Shin Megami Tensei V in-game graphics are all white and green when using Vulkan backend.  Note game menu graphics are normal. Tried both 472.12 and latest nvidia drivers. Tried various settings with no effect on colors. Colors only render correctly with OpenGL.

System is Win10, 6700K with nvidia 1080.

Note: using SMTV with latest update 1.0.2.

Here is a screenshot and log:

![010063b012dc6000_2021-12-11_17-14-45-464](https://user-images.githubusercontent.com/27729732/145693242-12c6050b-6bfc-484b-9f86-9293d0313369.png)

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7697818/yuzu_log.txt)

**Comment by gidoly** [comment]
try clear shader cache

**Comment by tapperp** [comment]
> try clear shader cache

Yes, clearing shader cache does help a lot. Most colors are normal now, but there is still a problem with weird green glowiness all over.

Here are screens and log.
![010063b012dc6000_2021-12-11_21-20-00-917](https://user-images.githubusercontent.com/27729732/145697974-adaefddd-cd4f-487d-87bb-097c865c00a6.png)
![010063b012dc6000_2021-12-11_21-20-51-506](https://user-images.githubusercontent.com/27729732/145697976-d092a5f0-c07b-4dd7-96f1-6eaf4a8f061b.png)

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7698026/yuzu_log.txt)

**Comment by tapperp** [comment]
I tried turning off the 60fps mod, deleted shader cache, and set resolution to 1x. Still incorrect colors.

Screenshots and log:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7698029/yuzu_log.txt)
![010063b012dc6000_2021-12-11_21-29-57-965](https://user-images.githubusercontent.com/27729732/145698130-eef0f18e-822e-4713-80b1-fe586894dd20.png)
![010063b012dc6000_2021-12-11_21-30-07-162](https://user-images.githubusercontent.com/27729732/145698133-3047517c-2fdf-4f77-8812-2f0b2ba808a8.png)
![010063b012dc6000_2021-12-11_21-30-19-420](https://user-images.githubusercontent.com/27729732/145698135-3feaa354-34e7-4f58-ac1c-8a3d93250411.png)

**Comment by tapperp** [comment]
NOTE: tried on another PC with 8th gen i7 and nvidia 2070, and after some shaders were generated, on the second launch the visuals were correct.

On the first PC with i7 6700K and nvidia 1080, the visuals are always wrong to some degree, primarily in regards to weird green artifacts. Below is the most recent screenshot - it is all white and green again now. This occurred after deleting shader cache and doing a second run to generate some cache first. Yuzu used to run this game fine on this PC until recently.

![010063b012dc6000_2021-12-11_23-26-26-870](https://user-images.githubusercontent.com/27729732/145700270-b86dfbeb-50e1-4942-80c1-0f676cb718a5.png)

**Comment by rujiel** [comment]
Per the last screenshot--this happens occasionally for me when the game starts, but it goes away if I go to the world of shadows and then back.

**Comment by Phraqzy** [comment]
This also happens with AMD. In my case E5-2640v3 with RX 580 using Mesa 21.3.2

**Comment by dante3732** [comment]
Maybe lightning fixed on Early Access 2790.
